cypress/support/cypress-support/src/utils.js

```diff
@@ -83,6 +83,8 @@ function createIntentMessage(task, jsonParams, map = null) {
   map && map.hasOwnProperty(CONSTANTS.IS_NOT_SUPPORTED_API)
     ? (jsonQueryParams.isNotSupportedApi = map.isNotSupportedApi)
     : false;
+
```

Comment:
Can we make the timeout value configurable? - Fine with hardcoded default in code but have the ability to override it

Reply:
moved the timeout value to constants